src/plugins/lightbox/lightbox-en.hbs

```diff
@@ -403,7 +403,7 @@
 					</li>
 					<li>
 						<a href="demo/2_b.jpg" title="Image 2" data-title="image_2_b">
-							<img src="demo/2_s.jpg" alt="Image 2" aria-describedby="image2-desc" longdesc="#image2-desc" />
+							<img src="demo/2_s.jpg" alt="Image 2" aria-describedby="image2-desc" />
 						</a>
 					</li>
```

Comment:
I am looking for something like this or similar. The goal is to keep the link between the image and it's long description
```suggestion
							<img src="demo/2_s.jpg" alt="Image 2" aria-describedby="image2-desc" />
						</a>
						<p><a href="#image2-desc">View long description for image 2</a></p>
					</li>
```


```suggestion
							<img src="demo/2_s.jpg" alt="Image 2" aria-describedby="image2-desc" />
						</a>
						<details>
						   <summary>View long description for image 2</summary>
						   <p>Description here</p>
						</details>
					</li>
```

Reply:
I realized yesterday afternoon that the lightbox plugin re-uses image 2's ``img`` element (from the thumbnail links) and dynamically copies it into lightbox popups... So whatever we end up doing will require changes to the plugin itself. IMO that's goes way beyond the scope of this PR.

I'm going to mark this conversation as resolved since I've just updated the PR to exempt lightbox's ``longdesc`` attributes from the ``htmllint`` task.